opium/app.ml

```diff
@@ -83,8 +83,8 @@ let attach_middleware { verbose; debug; routes; middlewares; _ } =
   in
   [ Some (routes |> create_router |> Router.m) ]
   @ ListLabels.map ~f:Option.some middlewares
-  @ [ (if verbose then Some Debug.trace else None)
-    ; (if debug then Some Debug.debug else None)
+  @ [ (if verbose then Some Logger.m else None)
+    ; (if debug then Some (Opium_kernel.Middlewares.debugger ()) else None)
```

Comment:
We probably want to include `Opium_kernel.Middleware` in `Opium.Middleware` to avoid having to have a single entry point when building apps with `opium`.